data/coalition/coalition missions.txt

```diff
@@ -146,6 +146,77 @@ mission "Coalition License Patch"
 
 
 
+mission "lost in coalition"
+	landing
+	invisible
+	source
+		government "Coalition" "Heliarch"
+	to offer
+		has "Coalition: Contributor: done"
+	on offer
+		require "Jump Drive" 0
+		event "lost in coalition" 14
+		fail
+
+event "lost in coalition"
+
+
+
+mission "Coalition: Lost: Marker"
+	name "Foreign Aid"
+	description "The Heliarchs have noticed that you've lost your jump drive, being now stranded in Coalition space, and have offered to help. Head to <destination> to speak with them."
+	landing
+	source
+		near "Ekuarik" 1 100
+	destination "Ring of Wisdom"
+	to offer
+		has "event: lost in coalition"
+	on offer
+		conversation
+			`Another landing, another day spent stranded in this alien region, with no other humans in sight. Be it with the help of a Heliarch agent, via their own translation devices, or just by gesturing and hoping for the best, you've managed to get by just fine, but the thought that you may live the rest of your life living among an alien civilization refuses to leave your mind.`
```

Comment:
Given that the time that I lost my JD in Coalition space I hadn't even realized it until much later, I'm not so sure if this is how we should start this conversation. The 14 day wait in Remnant space given how small it is means that the player is more likely to realize their mistake, but in Coalition space it could take longer than 14 days for the player to realize.

I'd suggest getting rid of this first line and instead starting with the player being approached by the Heliarchs, who comment on the player's lack of a JD. Then, have a choice where the player can either thank the Heliarchs for their generosity, or be shocked that they've lost their JD.